apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.ts

```diff
@@ -198,10 +198,14 @@ export function useDeepLinkHandler() {
           const modal =
             url === "send" ? "MODAL_SEND" : url === "receive" ? "MODAL_RECEIVE" : "MODAL_DELEGATE";
           const { currency, recipient, amount } = query;
-          if (!currency || typeof currency !== "string") return;
+
           if (url === "delegate" && currency !== "tezos") return;
-          const c = findCryptoCurrencyByKeyword(currency.toUpperCase()) as Currency;
-          if (!c || c.type === "FiatCurrency") {
+
+          const c = findCryptoCurrencyByKeyword(
```

Comment:
I know it was named like this before but could you rename it to something more readable, please? Like `c` => `foundCurrency`

Reply:
yes :)